### PR TITLE
cli: support alternative names for setup-openshift-heketi-storage

### PR DESCRIPTION
**Pull request by phlogistonjohn**
This is an only partially serious suggestion triggered by a
comment @jarrpa made a while back.
(see
https://github.com/gluster/gluster-kubernetes/issues/434#issuecomment-366006365
)

Using cobra's aliases feature we no longer have to be embarrassed by
an overly openshift specific command name, but can now refer to it
via much more generic names. In the future we could even swap
the "true" name for one of these fine aliases.

### CODE REVIEW

**Comment by phlogistonjohn** [comment]
@nixpanic that's pretty nice. I'll keep that in mind for next time. Thanks!

File: client/cli/go/cmds/heketi_storage.go [hunk 1]
```diff
@@ -282,7 +282,11 @@ func createHeketiCopyJob(volume *api.VolumeInfoResponse) *batch.Job {
 }
 
 var setupHeketiStorageCommand = &cobra.Command{
-	Use:   "setup-openshift-heketi-storage",
+	Use: "setup-openshift-heketi-storage",
+	Aliases: []string{
+		"setup-heketi-self-storage",
```
Comment (obnoxxx) [inline client/cli/go/cmds/heketi_storage.go]: I am thinking how about "setup-heketi-db-storage" instead of "setup-heketi-self-storage"? ...

**Comment by phlogistonjohn** [comment]
1. Updated name to  "setup-heketi-db-storage" as suggested
2. Updated format of commit message

**Comment by jarrpa** [comment]
As the inspiration/instigator of this change, chiming in with an LGTM. :)

Also, it looks like the PR needs a rebase.

**Pull request merged**